src/quo2/components/loaders/skeleton_list/view.cljs

```diff
@@ -0,0 +1,104 @@
+  [rn/view
+   {:style (style/container content)}
+   [rn/view {:style (style/avatar color)}]
+   [rn/view {:style style/content-container}
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :message :author)
+               :index   index
+               :content content
+               :color   color})}]
+    [rn/view
+     {:style (style/content-view
+              {:type    (if (= content :list-items) :author :message)
+               :index   index
+               :content content
+               :color   color})}]
+    (when (= content :notifications)
+      [rn/view
+       {:style (style/content-view {:type    :message2
+                                    :index   index
+                                    :content content
+                                    :color   color})}])]])
+
+(defn- f-animated-skeleton-view
+  [{:keys [style color skeleton-height animated? translate-x window-width] :as data}]
+  (let [loading-color (colors/theme-colors colors/neutral-10 colors/neutral-60)]
+
+    (rn/use-effect
+     (fn []
+       #(when-not animated?
+          (reanimated/cancel-animation translate-x))
+       (reanimated/animate-shared-value-with-repeat
+        translate-x
+        window-width
+        1000
+        :linear
+        -1
+        false)
+       [animated?]))
+
+    [masked-view/masked-view
+     {:style       {:height skeleton-height}
+      :maskElement (reagent/as-element (static-skeleton-view data))}
+     [rn/view
+      {:style {:flex             1
+               :background-color color}}
+      [reanimated/linear-gradient
+       {:colors [color color loading-color color color]
+        :start  {:x 0 :y 0}
+        :end    {:x 1 :y 0}
+        :style  style}]]]))
+
+(defn- animated-skeleton-view
+  [props]
+  [:f> f-animated-skeleton-view props])
+
+(defn- f-internal-view
+  [{:keys [content theme blur? parent-height animated?] :as props}]
+  (let [{window-width :width}   (rn/get-window)
+        translate-x             (reanimated/use-shared-value (- window-width))
+        animated-gradient-style (reanimated/apply-animations-to-style
+                                 {:transform [{:translateX translate-x}]}
+                                 {:width  window-width
+                                  :height "100%"})
+        skeleton-height         (get-in constants/layout-dimensions [content :height])
+        number-of-skeletons     (int (Math/ceil (/ parent-height skeleton-height)))
+        color                   (cond
+                                  (and blur? (= theme :dark)) colors/white-opa-5
+                                  (= theme :dark)             colors/neutral-90
+                                  :else                       colors/neutral-5)
+        component               (if animated? animated-skeleton-view static-skeleton-view)]
+
+    [rn/view
+     {:style               {:padding 8}
+      :accessibility-label :skeleton-list}
+     (doall
```

Comment:
`doall` is unnecessary. It's only needed if the body of the macro has side-effects, like deref'ing an atom.